src/helm/benchmark/static/benchmarking.js

```diff
@@ -1186,17 +1186,23 @@ $(function () {
   //////////////////////////////////////////////////////////////////////////////
   const $main = $('#main');
   const $summary = $('#summary');
+  var runManifest = undefined;
   $.when(
     $.get('schema.yaml', {}, (response) => {
       const raw = jsyaml.load(response);
       console.log('schema', raw);
       schema = new Schema(raw);
     }),
-    $.get(summaryJsonUrl(suite), {}, (response) => {
+    $.get(summaryJsonUrl(release), {}, (response) => {
       console.log('summary', response);
       summary = response;
-      $summary.append(`${summary.suite} (last updated ${summary.date})`);
+      $summary.append(`${summary.release} (last updated ${summary.date})`);
     }),
+    $.getJSON(runManifestJsonUrl(release), {}, (response) => {
+      runManifest = response;
+    }).then(() => {}, () => {
+      runManifest = undefined;
+    })
```

Comment:
Note: This is a hacky way to make sure that the getJSON Deferred doesn't fail and cause failure of the entire "master" Deferred (which means `.when` just executes the failure callback instead of the success callback, and nothing executes).

There's probably a better way to do this; let me know if you have suggestions.

Reply:
I think this is fine.

Here's a possible alternative:

In summary, provide the `suites` and `releases` fields for the new mode, and only the `suite` (singular) field for reverse compatibility mode.

Then reorder the promises; instead of:

```
(schema, summary, run manifest) and then (main function)
```

do

```
(schema, summary) and then (run manifest) and then (main function)
```

Then the run manifest logic can just skip fetching the file if summary indicates that we are in reverse compatibility mode.